client/js/netflix/watchScreen.js

```diff
@@ -111,11 +103,13 @@ var mutationObserver = new MutationObserver(function(mutations) {
             const title = getTitle();
             
             // Get the cast details
-            // const movieDetails = getCastDetails(title);
-            const casts = getCastDetails1(title);
-            console.log("printing cast details ", casts);
-            console.log(typeof(casts));
-            addToWatchScreen(casts)
+            const response = getCastDetails(title).then(result => {
```

Comment:
if you want, you can use async await. This also works though

Reply:
I tried that.. Somehow I couldn't seamlessly replicate that. So I did it with promises. I hope it should also do the intended behavior. If I get some more time, I will check on that again